scripts/jest/preprocessor.js

```diff
@@ -45,7 +45,10 @@ var babelOptions = assign(
     blacklist: babelDefaultOptions.blacklist.concat('validation.react'),
     _moduleMap: fixModules(assign({}, babelDefaultOptions._moduleMap, require('fbjs/module-map'), {
       'React': 'react',
+      'reactComponentExpect': 'react/lib/reactComponentExpect',
+      'ReactDOM': 'react-dom',
       'ReactUpdates': 'react/lib/ReactUpdates',
+      'ReactTestUtils': 'react/lib/ReactTestUtils.js',
```

Comment:
the only reason why these three modules work right now is because node-haste1 looks into node_modules folders which node-hast2 doesn't do anymore (unless explicitly required).